tutorial/CodeTrans/deploy/xeon.md

```diff
@@ -1,269 +1,201 @@
-# Single node on-prem deployment with TGI on Xeon Scalable processors
+# Single node on-prem deployment on Xeon Scalable processors
 
-This deployment section covers the single-node on-prem deployment of the CodeTrans example with OPEA comps using the Text Generation service based on TGI. The solution demonstrates building a code translation service using the `mistralai/Mistral-7B-Instruct-v0.3` model deployed on Intel® Xeon® Scalable processors. To quickly learn about OPEA in just 5 minutes and set up the required hardware and software, please follow the instructions in the [Getting Started](../../../getting-started/README.md) section.
+This section covers single-node on-prem deployment of the CodeTrans example. It will show how to build an end-to-end code translation service with the `mistralai/Mistral-7B-Instruct-v0.3` model deployed on Intel® Xeon® Scalable processors. To quickly learn about OPEA and set up the required hardware and software, follow the instructions in the [Getting Started](../../../getting-started/README.md) section.
```

Comment:
Do we want to use the word "build" or rather "create" or "deploy"?
I prefer "deploy" and change "deployed" to "running"  to not be repetitive.